dandi/cli/cmd_delete.py

```diff
@@ -10,7 +10,11 @@
 @devel_debug_option()
 @map_to_click_exceptions
 def delete(paths, skip_missing, dandi_instance="dandi", devel_debug=False):
-    """ Delete Dandisets and assets from the server """
+    """Delete Dandisets and assets from the server
+
+    PATH could be a local path or a URL to an asset, directory, or an entire
+    dandiset.
```

Comment:
"Dandiset" should be capitalized, shouldn't it?

Reply:
For consistency with prior summary one -- sure... but it seems that we do not have consistency in general, but other commands use it lowercased.  So I will make it lower cased here as well and it would look like

```
Usage: dandi [OPTIONS] COMMAND [ARGS]...

  A client to support interactions with DANDI archive
  (http://dandiarchive.org).

  To see help for a specific command, run

      dandi COMMAND --help

  e.g. dandi upload --help

Options:
  --version
  -l, --log-level [DEBUG|INFO|WARNING|ERROR|CRITICAL]
                                  Log level (case insensitive).  May be
                                  specified as an integer.  [default: INFO]

  --pdb                           Fall into pdb if errors out
  --help                          Show this message and exit.

Commands:
  delete    Delete dandisets and assets from the server.
  digest    Calculate file digests
  download  Download a file or entire folder from DANDI
  ls        List .nwb files and dandisets metadata.
  organize  (Re)organize files according to the metadata.
  upload    Upload dandiset (files) to DANDI archive.
  validate  Validate files for NWB (and DANDI) compliance.

```